External/FEXCore/Source/Interface/Core/JIT/Arm64/JIT.cpp

```diff
@@ -739,9 +739,9 @@ void *Arm64JITCore::CompileCode(uint64_t Entry, [[maybe_unused]] FEXCore::IR::IR
   // X1-X3 = Temp
   // X4-r18 = RA
 
-  auto GuestEntry = GetCursorAddress<uint64_t>();
+  GuestEntry = GetCursorAddress<uint64_t>();
```

Comment:
`GuestEntry` is a Guest or Host pointer?

Reply:
Host pointer